src/main/rx/rx.c

```diff
@@ -228,24 +238,37 @@ bool rxIsReceivingSignal(void)
     return rxSignalReceived;
 }
 
+bool rxAreFlightChannelsValid(void)
+{
+    return rxFlightChannelsValid;
+}
 static bool isRxDataDriven(void) {
     return !(feature(FEATURE_RX_PARALLEL_PWM | FEATURE_RX_PPM));
 }
 
-void updateRx(uint32_t currentTime)
+static void resetRxSignalTimeoutIfNeeded(uint32_t currentTime)
```

Comment:
`testRxSignalTimeout`? or maybe `processRxSignalTimeout`

Timeout is not reset in this function